share/spice/iplookup/iplookup.js

```diff
@@ -30,7 +37,7 @@
             templates: {
                 group: 'text',
                 options: {
-                    content: api_result.ddg.blacklists.length ? Spice.iplookup.content : null,
```

Comment:
Thanks for the extra examples @moollaza ! I think you just wrote half a page of documentation :laughing:

Returning the empty array in this case is not what we want as the conditional expression will evaluate as true and the blacklists template will load. 

The API will sometimes return an empty array for the blacklists value, therefore we need to check the length before using it.

We could use `getProperty` then if true check the length, or just check if its not `undefined` and check length. Either way works.